tests/widgets_block.rs

```diff
@@ -237,6 +238,8 @@ fn widgets_block_title_alignment() {
             .unwrap();
 
         terminal.backend().assert_buffer(&expected);
+        let area = Rect::new(1, 0, 13, 2);
+        assert_buffer_eq!(Buffer::empty(area), Buffer::empty(area));
```

Comment:
Hmm, what exactly the point of this addition?

Reply:
Ha none :D. I think I was probably debugging my vscode autocomplete not working (or something like that), on a random area and included that by mistake. I'll push a removal of that in a moment.

Reply:
>I'll push a removal of that in a moment.
Done